web/src/components/FeedbackContainer.tsx

```diff
@@ -111,6 +101,9 @@ export const FeedbackContainer = (props: FeedbackContainerProps): ReactElement =
       const feedbackEndpoint = createFeedbackEndpoint(cmsApiBaseUrl)
       await feedbackEndpoint.request(feedbackData)
       setSendingStatus(SendingState.SUCCESS)
+      if (onSubmitted) {
+        onSubmitted(true)
+      }
```

Comment:
Why is the check for `onSubmitted` necessary?
```suggestion
      onSubmitted(true)
```

Reply:
Must have looked wrong first time, I thought `onSubmit` is always truthy